packages/core/src/tracing/trace.ts

```diff
@@ -156,7 +156,10 @@ export function startInactiveSpan(context: TransactionContext): Span | undefined
 
   const hub = getCurrentHub();
   const parentSpan = getActiveSpan();
-  return parentSpan ? parentSpan.startChild(ctx) : hub.startTransaction(ctx);
+  return parentSpan
+    ? parentSpan.startChild(ctx)
+    : // eslint-disable-next-line deprecation/deprecation
+      hub.startTransaction(ctx);
```

Comment:
not relevant for this PR but this makes me realize we'll probably need to create an internal `startTxn` utility for us as a replacement of `hub.startTransaction` when its removed as long as we're still sending spans in transacitons.

Reply:
jup, also had that thought - for now we'll still need some internal thing, but we can refactor this on the v8 branch then I'd say :)